src/templates/togepi/Togepi.js

```diff
@@ -0,0 +1,262 @@
+            </div>
+            <p className="text-xs">{x.issuer}</p>
+            <RichTextArea
+                className="mt-2"
+                value={x.summary}
+                readOnly
+            />
+        </div>
+    );
+
+    const Certificate = () => 
+        data.certifications &&
+        config.certifications.enable && (
+            <div>
+                <Heading title="CERTIFICATE" />
+                <DevideLine/>
+                {data.certifications.filter(x => x.enable).map(CertificationItem)}
+            </div>
+    );
+
+    const AwardItem = x => 
+    (
+        <div key={x.id} className="mb-2">
+            <div className="flex justify-between">
+                <h6 className="font-semibold">{x.title}</h6>
+                <p className="text-xs font-medium">{x.date}</p>
+            </div>
+            <p className="text-xs">{x.awarder}</p>
+            <RichTextArea
+                className="mt-2"
+                value={x.summary}
+                readOnly
+            />
+        </div>
+    );
+
+    const Awards = () => 
+        data.awards &&
+        config.awards.enable && (
+            <div>
+                <Heading title={config.awards.heading} />
+                <DevideLine/>
+                <div className="flex flex-col mb-2">
+                    {data.awards.filter(x => x.enable).map(AwardItem)}
+                </div>
+            </div>
+    );
+
+    const SkillItem = x => (
+        <span
+            key={x.id}
+            className="text-xs rounded-full px-3 py-1 font-medium my-2 mr-2"
+            style={{
+                backgroundColor: theme.colors.primary,
+                color: theme.colors.background,
+            }}
+        >
+            {x.skill}
+        </span>
+    );
+
+    const Skill = () => 
+        data.skills &&
+        config.skills.enable && (
+            <div>
+                <Heading title="SPECIAL SKILLS"/>
+                <DevideLine/>
+                <div className="mt-1 flex flex-wrap">{data.skills.map(SkillItem)}</div>
+            </div>
+    );
+    
+    const Summary = () =>
+        data.basics.summary &&
+        config.summary.enable && (
+            <div>
+                <Heading title="ABOUT ME" />
+                <DevideLine/>
+                <RichTextArea
+                    value={data.basics.summary}
+                    readOnly
+                />
+            </div>
+    );
+
+    return(
+        <div style={{fontFamily: theme.font.family}}>
+            <div className="grid grid-cols-12 mb-5">
+                <div className="col-span-1 mt-8 h-8" style={{background: theme.colors.primary}}/>
+                <div className="col-start-2 col-span-11 mt-8 text-white h-8" style={{backgroundColor: theme.colors.accent}}>
+                    <h1 className="ml-3 mt-1">RESUME</h1>
+                </div>
+            </div>
+            
+            <div style={{'padding':'0 40px'}}>
```

Comment:
padding at the bottom is needed. otherwise, the content of the about me(summary) will be cut out

Reply:
use class px- pb- here instead of inline styling